### DISCUSSION TITLE
FastHttpUser: Error "RetriesExceeded("url", 0, original =The read operation timed out)

### DISCUSSION THREAD

**Discussion by tvhtu** [locustio/locust#3016] [Q&A]
Hi, I'm trying FastHttpUser and get an error that `Error "RetriesExceeded("url", 0, original =The read operation timed out)`
This error is only displayed when I run with FastHttpUser in high load. I don't get this issue when running with HttpUser at the same number of user. Please help me to fix it.

**Comment by cyberw** [comment]
Are you also having very high response times? Maybe try increasing the timeouts? (network_timeout / connection_timeout attributes on your class)
